src/main/resources/data/dungeons/lang/en_us.json

```diff
@@ -127,6 +143,43 @@
   "flavor_text.dungeons.metal_melee_weapon.gram.2": "of this metal in their lives",
   "flavor_text.dungeons.metal_melee_weapon.gram.3": "let alone an entire blade.",
 
+  "flavor_text.dungeons.armor.unimpressive": "It is unimpressive.",
+  "flavor_text.dungeons.armor.time.1": "Time has taken its toll",
+  "flavor_text.dungeons.armor.time.2": "on this armor.",
+  "flavor_text.dungeons.armor.soldier.1": "It was most likely used",
+  "flavor_text.dungeons.armor.soldier.2": "by some long-forgotten",
+  "flavor_text.dungeons.armor.soldier.3": "army foot soldier.",
+  "flavor_text.dungeons.armor.goodenough.1": "I guess this is better",
+  "flavor_text.dungeons.armor.goodenough.2": "than nothing?",
+  "flavor_text.dungeons.armor.goodenough.3": "- Davig Goodenough",
+  "flavor_text.dungeons.armor.fragile.1": "Seems like even the",
+  "flavor_text.dungeons.armor.fragile.2": "smallest bug could",
+  "flavor_text.dungeons.armor.fragile.3": "put a dent in it.",
+  "flavor_text.dungeons.armor.will_do": "It will do, for now.",
+  "flavor_text.dungeons.armor.unknowledgeable.1": "Impressive to anyone who",
+  "flavor_text.dungeons.armor.unknowledgeable.2": "isn't knowledgeable about",
+  "flavor_text.dungeons.armor.unknowledgeable.3": "armors.",
+  "flavor_text.dungeons.armor.standard_issue.1": "It seems to have a standard",
+  "flavor_text.dungeons.armor.standard_issue.2": "issue for this kind of",
+  "flavor_text.dungeons.armor.standard_issue.3": "armors.",
+  "flavor_text.dungeons.armor.glow.1": "It almost seems to glow",
+  "flavor_text.dungeons.armor.glow.2": "softly in the dim light.",
+  "flavor_text.dungeons.armor.craftsmanship.1": "It is clear that the",
+  "flavor_text.dungeons.armor.craftsmanship.2": "craftsmanship is of quality.",
+  "flavor_text.dungeons.armor.skeleton.1": "Enough to make any self-respecting",
+  "flavor_text.dungeons.armor.skeleton.2": "skeleton think twice about",
+  "flavor_text.dungeons.armor.skeleton.3": "engaging in combat with you.",
+  "flavor_text.dungeons.armor.carvings.1": "The carvings are immaculate",
+  "flavor_text.dungeons.armor.carvings.2": "but indecipherable.",
+  "flavor_text.dungeons.armor.legendary": "The work of a legendary smith.",
+  "flavor_text.dungeons.armor.reflection.1": "This armor looks like",
+  "flavor_text.dungeons.armor.reflection.2": "a thousand mirrors combined.",
+  "flavor_text.dungeons.armor.expense.1": "No expense was spared in the",
+  "flavor_text.dungeons.armor.expense.2": "creation of this armor.",
+  "flavor_text.dungeons.armor.gram.1": "Many will never see a gram",
+  "flavor_text.dungeons.armor.gram.2": "of this metal in their lives",
+  "flavor_text.dungeons.armor.gram.3": "let alone an entire armor.",
```

Comment:
nit: it should be "an entire piece of armour" or "an entire suit [or set] of armour" for a full set